integration-tests/src/tests/client/benchmarks.rs

```diff
@@ -26,7 +26,7 @@ fn benchmark_large_chunk_production_time() {
     let mb = 1024usize.pow(2);
 
     let n_txes = 20;
-    let tx_size = if checked_feature!("stable", WitnessTransactionLimits, PROTOCOL_VERSION) {
+    let tx_size = if checked_feature!("stable", StatelessValidationV0, PROTOCOL_VERSION) {
```

Comment:
Since the `PROTOCOL_VERSION` in the latest binary always satisfies this condition (since this is hard-coded version), can we replace this check with `true`? (also in other places in code)

Reply:
I think that is technically possible but I have mixed feelings about this. It also serves as a bit of documentation.

Reply:
Keeping as is for now